packages/mock-account-provider/app/entry.server.tsx

```diff
@@ -3,6 +3,23 @@ import type { EntryContext } from '@remix-run/node'
 import { Response } from '@remix-run/node'
 import { RemixServer } from '@remix-run/react'
 import { renderToPipeableStream } from 'react-dom/server'
+import { runSeed } from '../src/run_seed'
+
+declare global {
+  var seeded: boolean
+}
+
+if (!global.seeded) {
+  runSeed()
+    .then(() => {
+      global.seeded = true
+    })
+    .catch((e) => {
+      console.log(
+        `seeding failed with ${e}. If seeding has already completed this can probably be ignored`
+      )
+    })
+}
```

Comment:
This is different from what was in the example https://github.com/remix-run/blues-stack/blob/main/app/db.server.ts .

as far as I can see that _always_ re-runs when in production mode. I don't think we want that, so I went with a simpler test that only runs if the global is undefined. If we get the kind of reboot that erases the global, then we will log an error on restarts but not block.